plutus-core/plutus-ir/test/transform/letFloatIn/avoid-floating-into-RHS.golden

```diff
@@ -0,0 +1,25 @@
+(let
+  (nonrec)
+  (termbind
+    (strict)
+    (vardecl addInteger (fun (con integer) (fun (con integer) (con integer))))
+    (let
```

Comment:
Unless you want to demonstrate _this_ floating, but I don't think that you do.

Reply:
Yeah, this was generated from PlutusTx, and can be simplified manually